build/tasks/make_html_index.js

```diff
@@ -55,6 +55,10 @@ module.exports = function(grunt) {
 
     mainsrc += '</head>\n';
     mainsrc += '<body>\n';
+
+    // Div where addons embedding code live.
+    mainsrc += '<div id="addons"></div>';
```

Comment:
This task is supposed to be generic, so we should not have this here. Can't we create the 'div' manually if it does not exist (in the code that creates the sub-elements)?